base_image.dockerfile

```diff
@@ -24,20 +24,22 @@ ADD "https://api.github.com/repos/amordahl/ecstatic/commits?per_page=1" latest_c
 RUN git pull
 
 FROM python-build AS delta-debugger-build
-
+ENV DELTA_DEBUGGER_HOME=/SADeltaDebugger
 WORKDIR /
 RUN git config --global core.eol lf && \
  git config --global core.autocrlf input
 
+ADD "https://api.github.com/repos/pancax/SADeltaDebugger/commits?per_page=1" latest_debug
 RUN git clone https://github.com/Pancax/SADeltaDebugger.git
-RUN cd SADeltaDebugger/ProjectLineCounter &&  mvn install && \
-    cd ../ViolationDeltaDebugger && mvn package -DskipTests
-
+WORKDIR /SADeltaDebugger
+RUN git pull
```

Comment:
```suggestion

```
this is redundant. Adding the call to the github api will make sure you're always pulling the  latest version.